types/index.d.ts

```diff
@@ -183,6 +187,138 @@ declare module '@azure/functions' {
+         * @default false
+         */
+        enableContentNegotiation?: boolean;
+    }
+    /**
+     * Http response object and methods. Provided to your function when using HTTP triggers.
+     */
+    export interface HttpResponseApi extends IResponse {
+        /**
+         * Sets the HTTP response status code
+         * @param statusCode the status code to set
+         * @returns the updated HttpResponseApi instance
+         */
+        status: (statusCode: number | string) => HttpResponseApi;
+        /**
+         * Sets a particular header field to a value
+         * @param field the name of the header field to set
+         * @param val the value of the header field
+         * @returns the updated HttpResponseApi instance
+         */
+        setHeader(field: string, val: any): HttpResponseApi;
+        /**
+         * Has the same functionality as setHeader.
+         * Sets a particular header field to a value
+         * @param field the name of the header field to set
+         * @param val the value of the header field
+         * @returns the updated HttpResponseApi instance
+         */
+        header(field: string, val: any): HttpResponseApi;
+        /**
+         * Has the same functionality as setHeader.
+         * Sets a particular header field to a value
+         * @param field the name of the header field to set
+         * @param val the value of the header field
+         * @returns the updated HttpResponseApi instance
+         */
+        set(field: string, val: any): HttpResponseApi;
+        /**
+         * Get the value of a particular header field
+         * @param field the name of the header field to get
+         */
+        getHeader(field: string): any;
+        /**
+         * Has the same functionality as getHeader
+         * Get the value of a particular header field
+         * @param field the name of the header field to get
+         */
+        get(field: string): any;
+        /**
+         * Removes a particular header field
+         * @param field the name of the header field to remove
+         * @returns the updated HttpResponseApi instance
+         */
+        removeHeader(field: string): HttpResponseApi;
+        /**
+         * Set the 'Content-Type' header to a particular value
+         * @param type the value to set header 'Content-Type' to
+         * @returns the updated HttpResponseApi instance
+         */
+        type(type: string): HttpResponseApi;
+        /**
+         * Automatically sets the content-type then calls context.done()
+         * @param body (optional) body content to send
+         * @returns updated HttpResponseApi instance
+         * @deprecated this method calls context.done() which is deprecated, use async/await and pass the response as the return value instead.
+         * See the docs here for more information: https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node?tabs=v2#use-async-and-await
```

Comment:
Done. I wasn't sure what to name the aka.ms link, so feel free to suggest alternate names

Reply:
I don't care about the name - usually I just use the generated url. The reason I wanted an aka.ms link was so that we can change the destination url at any time without having to re-release the types package.

Reply:
Yeah makes sense :)